modules/plugin/imagemosaic/pom.xml

```diff
@@ -207,20 +207,4 @@
   </dependencies>
 
 
-  <build>
```

Comment:
I was not sure about this one, just noticed it makes the build slower and harder to debug